ep/lh/ep-lh-0009.md

```diff
@@ -0,0 +1,140 @@
+# linterhub: lhman
+
+| Metadata     |                                         |
+| ------------ |-----------------------------------------|
+| EP           | ep-lh-0009                              |
+| Version      | 1                                       |
+| Title        | linterhub: lhman                        |
+| Authors      | binore                                  |
+| Status       | Draft                                   |
+
+## Abstract
+
+This document describes the `lhman` - tool that allows to request engine's catalog, install engines and prepare necessary environment.
+
+## Motivation
+
+linterhub is a catalog that contains the information necessary to install, run and process the result of the work of the `linter`. To start working, you need a utility that allows you to centrally install the linter from the existing catalog.
+
+## Proposal
+
+lhman is a utility that allows you to obtain the catalog of linters and install them. For the implementation of the functional lhman is divided into `strategies`.
+
+## Strategies
+
+lhman implements the following strategies:
+
+| Name         | Description |
+| -            | -           |
+| catalog      | Get engines catalog |
+| install      | Install engine (or multiple engines) |
+| help         | Show help   |
+| version      | Get lhman version |
+
+> By default, space is used as delimiter between argument and value. The equal sign is also admissible. If the argument is at the same time several values, then it is recommended to use the comma sign to separate them. In addition, multiple declaration of arguments is allowed as an alternative. Example: `lhman strategy --arg1=value1,value2 --arg2 value1 --arg2=value2`
+
+### catalog
+
+This strategy generates 
+
+| Argument    | Type     | Required | Description |
+| -           | :-:      | :-:      | -           |
+| filter      | string   | -        | Engine name (or engine id) to install [ep-lh-0002](#related) |
+| keys        | string   | -        | Engine version (latest version by default) |
```

Comment:
BTW maybe let's exclude filters for now? We may implement it in the next version.

Reply:
5 days since the last change in EP, how many days we need to implement this feature? 
<img width="425" alt="screen shot 2018-04-10 at 09 11 23" src="https://user-images.githubusercontent.com/674129/38539065-3848f410-3c9f-11e8-9e96-1d711e56c1ba.png">